tests/Admin/AdminHelperTest.php

```diff
@@ -155,6 +161,107 @@ public function testItThrowsExceptionWhenDoesNotFindTheFullPath(): void
         $this->helper->getElementAccessPath($path, $object);
     }
 
+    public function testAppendFormFieldElement(): void
+    {
+        $container = $this->createMock(ContainerInterface::class);
```

Comment:
Not a blocker, but this method is very long. It would help if you could extract some methods (e.g. prepareAdmin, assertForm, ...), so the test is more readable.

Reply:
Let’s do it in another PR and go forward here, it’s blocking a real project 👍🏻